apps/api/v1/lib/validations/shared/queryExpandRelations.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+
+const expandEnum = z.enum(["team"]);
```

Comment:
We can add more relations here, as we wish to allow expansion via expand parameter

Reply:
?expand=team&expand=.. Have we considered a more industry standard param like `select=`? (Not a blocker just curious)